src/os/Journal.h

```diff
@@ -71,6 +72,8 @@ class Journal {
 
   virtual bool should_commit_now() = 0;
 
+  virtual int _op_journal_transactions_prepare(list<ObjectStore::Transaction*>& tls, bufferlist& tbl) = 0;
```

Comment:
can we call this prepare_entry() so that it looks like submit_entry (they are now coupled)?